File: Resources/Script Group/Base64 Encode.js
importPackage(Packages.java.util);
importPackage(Packages.java.nio.charset);

function invoke(msg)
{
    var data = msg.get("data");
    var base64EncodedData = Base64.getEncoder().encodeToString(data.getBytes(StandardCharsets.UTF_8));
    msg.put("data.base64", base64EncodedData);
    return true;
}

File: Resources/Script Group/Base64 Encode.py
from java.util import Base64
from java.lang import String
from java.nio.charset import StandardCharsets


def invoke(msg):
    data = msg.get("data")
    if data is not None:
        dataEncoded = Base64.getEncoder().encodeToString(String(data).getBytes(StandardCharsets.UTF_8))
        msg.put("data.base64", dataEncoded)
    return True